### DISCUSSION TITLE
localhost redirect too many times, try clearing your cookie

### DISCUSSION THREAD

**Discussion by helmihasibuan** [codeigniter4/shield#487] [Q&A] [answered]
i try to log in, but after insert email and password it should redirect to 2fa. but before sending 2fa i got this problem "localhost redirect too many times, try clearing your cookie". 

i write  `'session' => ['except' => ['login', 'register']] `  on global filters, and then that problem showing.
if i remove that code, it back to normal. 

any other solution to login and register?

thank you

![image](https://user-images.githubusercontent.com/65268735/198755803-db255b75-f037-40ec-8e1f-06845b21a136.png)

**Comment by datamweb** [comment, marked as answer]
Hello @helmihasibuan  ,
Thank you for your report.

Please proceed as follows.
```php 
 'session' => ['except' => ['login*', 'register*', 'auth*']],
```
If the problem is solved, please let me know.

**Reply by helmihasibuan** [reply]
thanks a lot @datamweb. it works

**Reply by datamweb** [reply]
I sent a PR to correct the documents. #488
